Libraries/LibWeb/WebAssembly/WebAssembly.cpp

```diff
@@ -134,7 +134,7 @@ WebIDL::ExceptionOr<GC::Ref<WebIDL::Promise>> compile(JS::VM& vm, GC::Root<WebID
 }
 
 // https://webassembly.github.io/spec/web-api/index.html#dom-webassembly-compilestreaming
-WebIDL::ExceptionOr<GC::Ref<WebIDL::Promise>> compile_streaming(JS::VM& vm, GC::Root<WebIDL::Promise> source)
+WebIDL::ExceptionOr<GC::Ref<WebIDL::Promise>> compile_streaming(JS::VM& vm, GC::Root<WebIDL::Promise> const& source)
```

Comment:
GC::Root is essentially a pointer, not sure why you'd want to pass it as const&

Reply:
Root is a bit more than a pointer, as opposed to GC::Ref/GC::Ptr that are free to copy.

Reply:
Ah, my bad then.

Reply:
I undid the changes to all `GC::Root`s before I saw the follow-up. I'm not sure if there's much of a performance gain passing these by const reference anyway.